### PR TITLE
CT-2342: Integrate /v1/vms/{videoManagerId}/analytics/videos endpoint to retrieve number of plays

### PR DESCRIPTION
**Pull request by jschaedl**

### CODE REVIEW

File: lib/ApiClient/AbstractApiClient.php [hunk 1]
```diff
@@ -262,6 +262,25 @@ public function getVideo(int $videoManagerId, string $videoId, ?VideoRequestPara
...
+    public function getPlays(int $videoManagerId, string $videoId): int
+    {
+        $options = [
+            self::OPT_VIDEO_MANAGER_ID => $videoManagerId,
+            'videoIds' => [
+                $videoId
+            ]
+        ];
+
+        $response = $this->makeRequest('GET', 'analytics/videos', $options);
```
Comment (jschaedl) [inline lib/ApiClient/AbstractApiClient.php:274]: See https://developers.movingimage.com/#/Video%20Analytics/get_v1_vms__videoManagerId__analytics_videos

**Comment by jschaedl** [comment]
Hi @pavbis & @BohdanKyryliuk, could you please have a look. Thank you.

**Review by pavbis — COMMENTED** [review]
@jschaedl I don't think you need to fork the repo. Someone from 300 probably has to add you to this GH account so you are a member of that mi24 stuff and can open prs directly in the repo.

**Review by BohdanKyryliuk — APPROVED** [review]
LGTM!

**Pull request closed without merging**